README.md

```diff
@@ -97,9 +104,16 @@ Otherwise, the following steps will work for any language:
 
 ## Detailed usage
 
-### Enhancing streaming video with asbplayer features
+First, install the Chrome [extension](https://chromewebstore.google.com/detail/asbplayer-language-learni/hkledmpjpaehamkiehglnbelcpdflcab).
+
+In order to make use of any of asbplayer's features (those outside of subtitles, e.g. mining), subtitles must first be loaded into the extension, with the context of loading them into streaming video or with local video files. The following section instructs how to do this.
+
+---
+### Subtitle setup and features
 
-Install the Chrome [extension](https://chromewebstore.google.com/detail/asbplayer-language-learni/hkledmpjpaehamkiehglnbelcpdflcab). There are a number of ways to enhance streaming video with asbplayer:
+### Injecting and detecting subtitles into streaming video

```

Comment:
For the sections under `Subtitle setup and features` I would use smaller headings to indicate that each subsection belongs to this larger section. So use `####` instead of `###`. Similar for other subsections.

To be consistent with the section below about local files I would say:

> ### Enhancing streaming video with asbplayer-controlled subtitles